dapps/digital-twin-detail/src/components/sharing/SharingOverview.vue

```diff
@@ -94,7 +94,15 @@
           </template>
 
           <template v-slot:cell(containers)="data">
-            {{ data.item.containerNames.join(', ') }}
+            <!--TODO:  :title="data.item[containerName].fields.join(', ')" -->
+            <span
+              v-for="(containerName, idx) in data.item.containerNames"
+              :key="containerName"
+              :title="data.item.containerFields[data.item.containers[idx]]"
+            >
+              {{ (containerName !='' && idx != 0) ? ', ' : '' }}
```

Comment:
This is rather ugly to read in a template. Refactoring it into a helper function would also help with the TODO above :)

Reply:
this is actually the TODO above, just forgot too remove it ;) 
I can't think of any helper function to do this properly. We would rather need a new component.

Reply:
OK, then at least this :p 
```suggestion
              {{ (containerName != '' && idx != 0) ? ', ' : '' }}
```